### ISSUE TITLE
Is it still working ? 

### ISSUE THREAD

**Issue by gordielachance** [dfelinto/image-swapper#2]
Hi, 

I use Blender 2.79 and when I enable your addon, I can't see the new panel under the texture tab.
Is this addon still working ?
Thanks !

**Comment by dfelinto** [comment]
Yes it is working. Maybe you forgot to enable the addon? I just tried here and it seems fine:

![image](https://user-images.githubusercontent.com/843498/41990501-d9b11a54-7a42-11e8-96d0-f061ecb29c0a.png)
